sdk/digitaltwins/azure-digitaltwins-core/API design.md

```diff
@@ -753,24 +753,15 @@ public <T> Mono<Response<T>> createDigitalTwinWithResponse(String digitalTwinId,
 /**
  * Deletes a digital twin. All relationships referencing the digital twin must already be deleted.
  *
- * @param digitalTwinId The id of the digital twin. The id is unique within the service and case sensitive.
+ * @param digitalTwinId The Id of the digital twin. The Id is unique within the service and case sensitive.
  */
 @ServiceMethod(returns = ReturnType.SINGLE)
 public Mono<Void> deleteDigitalTwin(String digitalTwinId)
 
 /**
  * Deletes a digital twin. All relationships referencing the digital twin must already be deleted.
  *
- * @param digitalTwinId The id of the digital twin. The id is unique within the service and case sensitive.
- * @param options The optional settings for this request
- */
-@ServiceMethod(returns = ReturnType.SINGLE)
-public Mono<Void> deleteDigitalTwin(String digitalTwinId, RequestOptions options)
-
-/**
- * Deletes a digital twin. All relationships referencing the digital twin must already be deleted.
- *
- * @param digitalTwinId The id of the digital twin. The id is unique within the service and case sensitive.
+ * @param digitalTwinId The Id of the digital twin. The Id is unique within the service and case sensitive.
```

Comment:
We do not need this since this is a max overload one and we have overloads with `Response<>`